miner/algo_common.go

```diff
@@ -301,7 +329,16 @@ func (envDiff *environmentDiff) commitBundle(bundle *types.SimulatedBundle, chDa
 
 	bundleProfit := coinbaseBalanceDelta
 
-	bundleActualEffGP := bundleProfit.Div(bundleProfit, big.NewInt(int64(gasUsed)))
+	var (
```

Comment:
I find `gasUsedBigInt := new(big.Int).SetUint64(gasUsed)` clearer than using var blocks, and it's the convention in the codebase

Reply:
the codebase is large, there are exceptions to the claim that var blocks are not the convention

see: execution.go, transition.go, runner.go, state_processor.go

Reply:
the var block doesn't make sense anymore since we no longer check for zero, moved back to existing convention